gen3-integration-tests/services/gen3workflows.py

```diff
@@ -0,0 +1,133 @@
+from typing import Dict, Tuple
+
+import boto3
+import botocore.exceptions
+import pytest
+import requests
+from gen3.auth import Gen3Auth
+from utils import logger
+
+
+class Gen3Workflow:
+    def __init__(self):
+        self.BASE_URL = f"{pytest.root_url}"
+        self.SERVICE_URL = "/workflows"
+        self.S3_ENDPOINT_URL = f"{self.BASE_URL}{self.SERVICE_URL}/s3"
+
+    def _get_access_token(self, user: str = "main_account") -> str:
+        """Helper function to retrieve an access token."""
+        auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
+        return auth.get_access_token()
+
+    def _get_s3_client(self, access_token: str):
+        """Creates and returns an S3 client."""
+        return boto3.client(
+            service_name="s3",
+            aws_access_key_id=access_token,
+            aws_secret_access_key="N/A",  # pragma: allowlist secret
+            endpoint_url=self.S3_ENDPOINT_URL,
+        )
+
+    def _get_bucket_and_key(self, object_path: str) -> Tuple[str, str]:
+        """Parses object_path into bucket and key."""
+        # cleaner than using the string.split() method
+        bucket, _, key = object_path.partition("/")
+        return bucket, key
+
+    def get_storage_info(self, user: str = "main_account", expected_status=200) -> Dict:
+        """Makes a GET request to the `/storage/info` endpoint."""
+        access_token = self._get_access_token(user) if user else None
+        storage_url = f"{self.BASE_URL}{self.SERVICE_URL}/storage/info"
+        headers = {
+            "Content-Type": "application/json",
+            "Authorization": f"bearer {access_token}",
+        }
```

Comment:
`"Authorization": "bearer None"` tests an invalid token, not an anonymous call. If the user is None, it would make more sense to send the request without the Authorization header.
Invalid tokens can be tested as well, but let's make it explicit in the tests. I'm not sure it's necessary since the unit tests should test that already.